onnx/numpy_helper.py

```diff
@@ -83,15 +83,17 @@ def from_array(arr, name=None):  # type: (np.ndarray[Any], Optional[Text]) -> Te
     if arr.dtype == np.object:
         # Special care for strings.
         tensor.data_type = mapping.NP_TYPE_TO_TENSOR_TYPE[arr.dtype]
-        for e in arr:
+        flat_array = arr.flatten()
```

Comment:
why do we need flatten here? We already handle the np.ndarray case already. (although, we probably should throw exception if it's 3d string array.

Reply:
@houseroad Current infrastructure does not handle strings input at all. I tried to avoid investing too much time in it. This line was introduced to accommodate another test Cast which I had to change as well.

Reply:
Since it's not handled appropriately, please add some comments on why flatten is called here, and leave a TODO mark here.